shell/platform/windows/platform_handler_unittests.cc

```diff
@@ -51,6 +61,8 @@ class TestPlatformHandler : public PlatformHandler {
   MOCK_METHOD2(SystemSoundPlay,
                void(const std::string&,
                     std::unique_ptr<MethodResult<rapidjson::Document>>));
+
+  // XXX delete this? FRIEND_TEST(PlatformHandler, ReleaseClipboard);
```

Comment:
Can this be deleted?

Reply:
Ouch! How embarrassing :) I really need to port my git alias over to my windows machine!

Reply:
No worries, I do the same thing all the time! Don't judge me when I do this too in a future pull request! 😂